src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,14 @@
-export const TodoList = () => {};
+import { TodoWithUser } from '../../react-app-env';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: TodoWithUser[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} key={todo.id} data-id={todo.id} />
```

Comment:
```suggestion
      <TodoInfo todo={todo} key={todo.id} />
```

There's no sense passing an id because we access an id in TodoInfo component from the **todo prop**